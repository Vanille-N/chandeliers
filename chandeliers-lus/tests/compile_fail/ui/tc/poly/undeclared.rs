//! Undeclared type variable on node.
chandeliers_lus::decl! {
    node id(t0: T) returns (t1: T);
    let t1 = t0; tel;
}

fn main() {}
