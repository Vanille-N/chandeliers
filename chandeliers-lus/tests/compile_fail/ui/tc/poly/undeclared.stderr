error[E0412]: cannot find type `T` in this scope
 --> tests/compile_fail/ui/tc/poly/undeclared.rs:2:13
  |
2 |     node id(t0: T) returns (t1: T);
  |             ^^ not found in this scope

error[E0412]: cannot find type `T` in this scope
 --> tests/compile_fail/ui/tc/poly/undeclared.rs:2:29
  |
2 |     node id(t0: T) returns (t1: T);
  |                             ^^ not found in this scope
